src/core/Clock.js

```diff
@@ -167,3 +177,18 @@ getJasmineRequireObj().Clock = function() {
 
   return Clock;
 };
+
+/**
+ * Stubs Node.js Timeout class
+ */
+function Timeout(id) {
```

Comment:
This leaks a `timeout` global when run in a browser.

Reply:
In addition to not making it a global, giving it a name like `StubTimeout` would also help with debugging.